thorlcr/thorutil/thbuf.cpp

```diff
@@ -2568,15 +2595,15 @@ class CSharedFullSpillingWriteAhead : public CInterfaceOf<ISharedRowStreamReader
             {
                 StringBuffer tracing;
                 getFileIOStats(tracing, iFileIO);
-                activity.ActPrintLog("CSharedFullSpillingWriteAhead: removing spill file: %s%s", iFile->queryFilename(), tracing.str());
+                activity.ActPrintLog("CSharedFullSpillingWriteAhead::outputStopped closing tempfile writer: %s %s", tempFileOwner->queryIFile().queryFilename(), tracing.str());
                 closeWriter();
-                iFile->remove();
+                tempFileOwner->noteSize(0);
```

Comment:
Has this actually deleted the file at this point?  Should there be a CFileOwner::remove() similar to the destructor (with protection for double removal?)